facebook/Docusaurus
# fix: render proper HTML instead of raw markdown in meta description

## Motivation

A pure markdown text was previously shown in "og:description" and "description" meta tags in Docs. Markdown text should be rendered into the HTML instead. Fixes [#1014](https://github.com/facebook/Docusaurus/issues/1014)

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/Docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

yes

## Test Plan

1. Create any doc page and start the application to render changes. 
2. Open a dev tools in your browser and look for <meta> tags in head of your doc page.

Test by Open Graph Check:

og:description **before changes**:

![before](https://user-images.githubusercontent.com/33039909/47376216-56b60800-d6f2-11e8-831e-4e3a92d625da.png)

og:description **after changes**:

![after](https://user-images.githubusercontent.com/33039909/47376218-59186200-d6f2-11e8-9cf2-8fc371e242a0.png)


You should see that content of description is rendered into the raw HTML in `<meta name"description" >` and `<meta property="og:description" >` tags.
